pmultiqc/modules/quantms/quantms.py

```diff
@@ -169,7 +167,44 @@ def __init__(self):
             self.draw_oversampling()
             self.draw_delta_mass()
 
-        self.parse_msstats_input()
+        # TODO what if multiple are found??
+        for msstats_input in self.find_log_files("quantms/msstats", filecontents=False):
+            self.msstats_input_path = os.path.join(msstats_input["root"], msstats_input["fn"])
+            msstats_input_valid = True
+            self.parse_msstats_input()
+
+        if not msstats_input_valid:
```

Comment:
@daichengxin it maybe better to add the spectrum psms tables not because of the msstats is available but with a variable. The idea should be added the protein table by default and only add the psms tables if msstats is not available and a variable in the configuration file as for it. Then, in `quantms` if spectral counting is executed, we actually pass that variable to the pmultiqc. 

This is important because is some cases, people may skip the msstats generation in the quantms then the pipeline will add the PSMs always, which is something not desireable.

Reply:
- msstats generation:  draw peptide and protein quantification from msstats input file
- no msstats and feature intensity:  draw protein quantification from mzTab
- no msstats and spectrum counting: draw protein quantification and psm table from mzTab

right? @ypriverol

Reply:
Yes!!!

Reply:
We can implement the second option `no msstats and feature intensity: draw protein quantification from mzTab` in the future with Protein and peptide tables from mzTab.